### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Peacefulkind**
**For the fourth project in my portfolio website, I will:**

Use media queries to make sure that my website displays correctly on desktop screens.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,712 @@
...
+}
+
+@media (min-width: 768px) {
+  body {
+    padding: 24px;
+  }
+
+  .top {
+    border-bottom-left-radius: 100px;
+  }
```
Comment (OyePriscilla) [inline style.css]: - [ ] Kindly ensure that this container as the same width with the body element as we have it in the figma template. 👍 

> Kindly take note to implement this for `About section` and `Contact section` as well.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,712 @@
...
+    gap: 24px;
+    border-top-right-radius: 100px;
+    width: 96%;
+  }
+
+  .container2 {
+    display: flex;
+    flex-direction: row;
+    border-top-right-radius: 100px;
+  }
```
Comment (OyePriscilla) [inline style.css]: - [ ] Please ensure that this container children are aligned properly according to the given template. 👍 

## Figma Design
langstack

File: style.css [hunk 3]
```diff
@@ -0,0 +1,712 @@
...
+  font-size: 28px;
+  margin: 10px 0;
+  transition: all 0.5s ease;
+}
+
+.section {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  background-image: url("images/mobile_bg.png");
```
Comment (OyePriscilla) [inline style.css:86]: - [ ] Kindly add the appropriate background image for the desktop version. 😄

File: index.html [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature that you need
+        built or a project that needs coding. I’d love to help with it
+      </p>
+      <form action="https://formspree.io/f/mvoyvrlg" method="POST" id="contactme">
+        <input type="text" id="name" name='name' maxlength="30" placeholder="Enter your name" required />
+        <input type="email" id="email" name="email" placeholder="Enter your email" required />
+        <small></small>
+        <textarea id='message' name="message" maxlength="500" placeholder="Type your message here" required></textarea>
```
Comment (OyePriscilla) [inline index.html:291]: - [ ] Please increase the `height` of this element so as to ensure great user interaction. 😄

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Peacefulkind   😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Peacefulkind, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**